run_scans.py

```diff
@@ -338,6 +338,84 @@ def trimChamberV3(args):
 
     return
 
+def iterTrim(args):
+    """
+    Launches a call of iterativeTrim.py
+    
+    args - object returned by argparse.ArgumentParser.parse_args() 
+    """
+    
+    startTime = datetime.datetime.now().strftime("%Y.%m.%d.%H.%M")
+    
+    # Determine number of OH's
+    cardName = getCardName(args.shelf,args.slot)
+    amcBoard = HwAMC(cardName, args.debug)
+    print('opened connection')
```

Comment:
nitpicking, but can you use double quotes here, as you use them everywhere else?

Reply:
done (this was just copied and pasted from the other functions)